src/main/java/de/tum/in/www1/artemis/service/dto/athena/TextExerciseDTO.java

```diff
@@ -4,12 +4,15 @@
 
 import jakarta.validation.constraints.NotNull;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+
 import de.tum.in.www1.artemis.domain.TextExercise;
 import de.tum.in.www1.artemis.service.dto.GradingCriterionDTO;
 
 /**
  * A DTO representing a TextExercise, for transferring data to Athena
  */
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
 public record TextExerciseDTO(long id, String title, double maxPoints, double bonusPoints, String gradingInstructions, List<GradingCriterionDTO> gradingCriteria,
         String problemStatement, String exampleSolution) implements ExerciseDTO {
```

Comment:
I think this change has caused a failing test:

de.tum.in.www1.artemis.service.connectors.athena.AthenaSubmissionSendingServiceTest#testSendTextSubmissionsSuccess

It tries to access the value of `$.exercise.gradingInstructions` (null -> now not included anymore) in the JSON. Can you take a look?